src/components/Announcement.tsx

```diff
@@ -51,18 +60,26 @@ export default function ({ redirectTo }: { redirectTo: string }) {
   const onPage = location.pathname === redirectTo
 
   return (
-    <div id="bottom-bar" className={announceWrapper(animate)}>
+    <div className={announceWrapper(animate)}>
```

Comment:
Why we needed an `id` here?

Reply:
Don't have any idea, it's not used anywhere in the code